types/openlayers/index.d.ts

```diff
@@ -13,28 +13,17 @@
 
 declare type GlobalObject = Object;
 
-/**
- * @namespace ol
- */
 declare module ol {
     /**
      * Error object thrown when an assertion failed. This is an ECMA-262 Error,
      * extended with a `code` property.
      * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error}
-     * @constructor
-     * @extends {Error}
-     * @implements {oli.AssertionError}
-     * @param {number} code Error code.
      */
     class AssertionError extends Error {
         /**
          * Error object thrown when an assertion failed. This is an ECMA-262 Error,
          * extended with a `code` property.
          * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error}
-         * @constructor
-         * @extends {Error}
-         * @implements {oli.AssertionError}
-         * @param {number} code Error code.
```

Comment:
Just the `{number}` could be removed here, for example.

Reply:
Yes, you are right, this information may useful. I will revert them.